### PR TITLE
Awesome-XGBoost page

### PR DESCRIPTION
**Pull request by phunterlau**
Awesome XGboost page as @tqchen has suggested. It is the first version which contains the recent winning solutions using xgboost, interviews, tutorials, as well as list of tools, awards, and predictive services powered by xgboost.

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -36,23 +42,61 @@ This is a list of short codes introducing different functionalities of xgboost p
...
+* XGBoost helps Alexandre Barachant (“Cat”) and Rafał Cycoń (“Dog”) to win (1st place) the [Grasp-and-Lift EEG Detection](https://www.kaggle.com/c/grasp-and-lift-eeg-detection). Check out the [interview from Kaggle](http://blog.kaggle.com/2015/10/12/grasp-and-lift-eeg-winners-interview-1st-place-cat-dog/).
+* XGBoost helps Halla Yang to win (2nd place) the [Recruit Coupon Purchase Prediction Challenge](https://www.kaggle.com/c/coupon-purchase-prediction). Check out the [interview from Kaggle](http://blog.kaggle.com/2015/10/21/recruit-coupon-purchase-winners-interview-2nd-place-halla-yang/).
+* XGBoost helps Owen Zhang to win (1st place) the [Avito Context Ad Clicks competition](https://www.kaggle.com/c/avito-context-ad-clicks). Check out the [interview from Kaggle](http://blog.kaggle.com/2015/08/26/avito-winners-interview-1st-place-owen-zhang/).
+* There are many other great winning solutions and interviews, but this list is [too small](https://en.wikipedia.org/wiki/Fermat%27s_Last_Theorem) to put all of them here. Please send pull requests if important ones appear.
+
+
+## List of Tutorials
```
Comment (tqchen) [inline demo/README.md]: Remove List of, simply do Tutorials

File: demo/README.md [hunk 2]
```diff
@@ -36,23 +42,61 @@ This is a list of short codes introducing different functionalities of xgboost p
...
+* XGBoost helps Vlad Mironov, Alexander Guschin to win (1st place) the [CERN LHCb experiment Flavour of Physics competition](https://www.kaggle.com/c/flavours-of-physics). Check out the [interview from Kaggle](http://blog.kaggle.com/2015/11/30/flavour-of-physics-technical-write-up-1st-place-go-polar-bears/).
+* XGBoost helps Josef Slavicek to win (3rd place) the [CERN LHCb experiment Flavour of Physics competition](https://www.kaggle.com/c/flavours-of-physics). Check out the [interview from Kaggle](http://blog.kaggle.com/2015/11/23/flavour-of-physics-winners-interview-3rd-place-josef-slavicek/).
+* XGBoost helps Mario Filho, Josef Feigl, Lucas, Gilberto to win (1st place) the [Caterpillar Tube Pricing competition](https://www.kaggle.com/c/caterpillar-tube-pricing). Check out the [interview from Kaggle](http://blog.kaggle.com/2015/09/22/caterpillar-winners-interview-1st-place-gilberto-josef-leustagos-mario/).
```
Comment (tqchen) [inline demo/README.md]: Remove XGBoost, but simply add links to the solutions

File: demo/README.md [hunk 3]
```diff
@@ -1,13 +1,19 @@
-XGBoost Code Examples
-=====================
-This folder contains all the code examples using xgboost.
+#Awesome XGBoost 
+
+Welcome to the wonderland of XGBoost. This page contains a curated list of awesome XGBoost examples, tutorials and blogs. It is inspired by [awesom-MXnet](https://github.com/dmlc/mxnet/blob/master/example/README.md), [awesome-php](https://github.com/ziadoz/awesome-php) and [awesome-machine-learning](https://github.com/josephmisiti/awesome-machine-learning).
+
+## Contributing
```
Comment (tqchen) [inline demo/README.md]: Let us add a table of content here

**Comment by phunterlau** [comment]
added TOC generated by doctoc, plus some simplified text in the solution section

**Comment by phunterlau** [comment]
em....weird, `travis-ci` fails on testing `python_test` but this PR only changes a `.md` file @tqchen Can you have a look? Thanks.

**Comment by Far0n** [comment]
That will become a very huge page ^^

**Pull request merged**